project/GrasshopperBuild.scala

```diff
@@ -18,7 +18,7 @@ object BuildSettings {
       organization  := buildOrganization,
       version       := buildVersion,
       scalaVersion  := buildScalaVersion,
-      wartremoverWarnings ++= Warts.allBut(Wart.NoNeedForMonad, Wart.NonUnitStatements),
+      //wartremoverWarnings ++= Warts.allBut(Wart.NoNeedForMonad, Wart.NonUnitStatements),
```

Comment:
It's producing what I consider a lot of false positives, creating a lot of noise. I want to revisit this and dig deeper, but yes, I commented out on purpose